src/components/PetsList.jsx

```diff
@@ -1,19 +1,87 @@
 import FilterList from "./FilterList";
 import PetCard from "./PetCard";
 import { PetItems } from "../files/PetsData";
-import React from "react";
+import React, { useState } from "react";
+import CategoriesFilterCollapse from "./CategoriesFilterCollapse";
 
 const PetsList = () => {
+  const [filteredItems, setFilteredItems] = useState([]);
+  const [isFiltering, setIsFiltering] = useState(false);
+  const [filterItems, setFilterItems] = useState([
+    { type: "species", name: "Cat", checked: true },
```

Comment:
each of these should be unchecked by default.

Reply:
done